sceptre/cli/list.py

```diff
@@ -89,6 +93,39 @@ def list_outputs(ctx, path, export):
                         ),
                         "text",
                     )
+
+    # Format outputs as !stack_output references.
+    elif export == "stackoutput":
+        for response in responses:
+            for stack_name, stack in response.items():
+                for output in stack:
+                    write(
+                        "!stack_output {0}.yaml::{1} [{2}]".format(
```

Comment:
what are your thoughts about not including `!stack_output` or `!stack_ouput_external` in the output?

Reply:
I have tried to be consistent with the original design as much as possible, so since `--export=envvar` prints out Bash code that can be copied/pasted (or eval'ed) this also prints out formatted output that can be copy/pasted into your code. So that's consistent and also convenient in that it saves some keystrokes of typing.

Similarly, I felt that following the case convention (`stackoutput` and `envvar` instead of `stack_output` and `env_var`) will look cleaner and more consistent in the docs and help message. 

But I don't really mind.

Reply:
Quoting @jfalkenstein offline:

> I think your argument is consistent. On a scale of caring 1-10... I'm at like a 2, though.

Shall we leave it all as-is then @zaro0508 ?

Reply:
My name's Jon Falkenstein and I approve this message.

Reply:
ok, leave this one as as-is is fine.